livecheck/main.py

```diff
@@ -118,12 +148,13 @@ def log_unhandled_github_package(catpkg: str):
 def get_props(search_dir: str,
               settings: LivecheckSettings,
               names: Sequence[str] | None = None,
-              exclude: Sequence[str] | None = None) -> Iterator[PropTuple]:
+              exclude: Sequence[str] | None = None,
+              progress: bool = False) -> Iterator[PropTuple]:
```

Comment:
Should be behind `*, ` to make it always require the keyword on the calling side.

Reply:
Lo siento , no lo entiendo

Reply:
Put the arguments like so: `(exclude: Sequence[str] | None = None, *, progress: bool = False)`.

The `*` means that on the calling side the keyword must always be present.

Wrong: `get_props(settings, names, exclude, True)`

Correct:  `get_props(settings, names, exclude, progress=True)`

With the correct way above it is much easier to determine what the boolean argument does.

Reply:
https://docs.python.org/es/3/tutorial/controlflow.html#keyword-only-arguments